test/component/will/testWillCodicils.js

```diff
@@ -34,11 +34,19 @@ describe('will-codicils', () => {
             testWrapper.testErrors(done, data, 'required', []);
         });
 
-        it(`test it redirects to death certificate page: ${expectedNextUrlForDeathCertificate}`, (done) => {
+        it(`test it redirects to death certificate page: ${expectedNextUrlForTasklist}`, (done) => {
```

Comment:
Please update this test description  as it now tests the redirect to the task list not the death certificate page